package-system/assimp/0001-O3DE-fixes-applied-on-top-of-376e46887780e5090af0198.patch

```diff
@@ -0,0 +1,317 @@
+From a8f0333acd438ce3038b64245a004f0f8a049f25 Mon Sep 17 00:00:00 2001
+From: AMZN-stankowi <4838196+AMZN-stankowi@users.noreply.github.com>
+Date: Tue, 7 Dec 2021 07:32:09 -0800
+Subject: [PATCH] O3DE fixes applied on top of
+ 376e46887780e5090af0198e4e5577446159104d, the base version O3DE started with.
+ mesh.h - added texture coordinate names material.h - added PBR material
+ properties- these have already been submitted to a later revision of assimp
+ FBXProperties.cpp - added support for properties with capital F Float, we had
+ some FBX files coming in with those. FBXConverter.cpp - this is the majority
+ of our changes. A lot of this is fixing how the transform hierarchy is
+ calculated to better preserve the default pose. The texture coordinate names
+ support is here, PBR materials, and a crash on keyframe list population is
+ downgraded to an error instead. AssxmlFileWriter.cpp - texture coordinate
+ names are added to the output here.
+
+Signed-off-by: AMZN-stankowi <4838196+AMZN-stankowi@users.noreply.github.com>
+---
+ code/AssetLib/Assxml/AssxmlFileWriter.cpp |   7 +-
+ code/AssetLib/FBX/FBXConverter.cpp        | 142 ++++++++++++++++------
+ code/AssetLib/FBX/FBXProperties.cpp       |   2 +-
+ include/assimp/material.h                 |  12 ++
+ include/assimp/mesh.h                     |   2 +
+ 5 files changed, 125 insertions(+), 40 deletions(-)
```

Comment:
Why is this in the patch?

Reply:
I just placed the Perforce copy of assimp on top of the github commit, and generated a patch. Is there a better way to do it?